src/components/SkeletonPage/SkeletonPage.tsx

```diff
@@ -101,14 +106,22 @@ function renderSecondaryActions(actionCount: number) {
   return <div className={styles.Actions}>{actions}</div>;
 }
 
-function renderTitle(title: string) {
-  const titleContent =
-    title === '' ? (
-      <SkeletonDisplayText size="large" />
+function SkeletonPageTitle({
+  title = '',
+  newDesignLanguage = false,
+}: {
+  title?: string;
+  newDesignLanguage?: boolean;
+}) {
+  if (title) {
+    return newDesignLanguage ? (
+      <h1 className={styles.newDesignLanguageTitle}>{title}</h1>
     ) : (
       <DisplayText size="large" element="h1">
-        {title}
+        <TextStyle variation="strong">{title}</TextStyle>
       </DisplayText>
     );
-  return <div className={styles.Title}>{titleContent}</div>;
+  }
+
+  return <SkeletonDisplayText size="large" />;
```

Comment:
Does this `<SkeletonDisplayText size="large">` have the same visual height as the new design language title at all viewport sizes?

I've not confirmed this but I'm guessing it doesn't as `<DisplayText size="large">` matches `<SkeletonDisplayText size="large">` and wrote our own logic instead of using `<DisplayText size="large">` in Page. (Alternatively `<SkeletonDisplayText size="large">` and thus `<DisplayText size="large">` matches the new header css in Page in which case we should just use DisplayText instead of writing custom css in Page)

We should make sure this Skeleton content is the same height so we don't trigger layout shifts here too.

Reply:
I have the same concern too, it looks like this change would only match what we have in `Title` if the `title` was passed in to skeleton page. Otherwise you'd always be getting display text which I don't think matches perfectly.

Reply:
Some of the gifs added to the PR description show the transition from Skeleton Display Text to Display Text:

![skeleton-page-title-null-mobile](https://user-images.githubusercontent.com/8534230/96027996-be594680-0e26-11eb-97c2-0f92ad63db27.gif)

![skeleton-page-title-null-wide](https://user-images.githubusercontent.com/8534230/96028071-d4670700-0e26-11eb-8dac-9fe8d1c61008.gif)

Reply:
Looks like `<SkeletonDisplayText size="large">` uses the `display-large` line heights for its heights - [28px base, 32px when-typography-not-condensed](https://github.com/Shopify/polaris-react/blob/e290a8c751db794cc35916698402364c16588aa5/src/styles/foundation/_typography.scss#L44-L47), while Page title / newDesignLanguageTitle uses 28px at all sizes) so these are indeed different.

That looks about right with the jump seen in those gifs.

Would you be interested in fixing this up so the Skeleton height matches real content so we don't get page jumps there? (uugh it looks like we currently get a jump in that first gif because the actions are taller than the title but that feels slightly out of scope for this ticket).

A div with a class like this should do the job (written but not tested):

```
// duplicated from SkeletonDisplayText.scss
$skeleton-display-text-max-width: rem(120px);

.newDesignLanguageSkeletonTitle {
  @include skeleton-shimmer;
  @include skeleton-content;

  max-width: $skeleton-display-text-max-width;
  height: rem(28px);
}
```